src/pages/understand-stacks/managing-accounts.md

```diff
@@ -106,24 +106,30 @@ The `balance` property represents the Stacks token balance, as hex-encoded strin
 Proofs, provided as hex-encoded strings, can be removed from the responses by setting the `proof` parameter:
 
 ```js
-const accountInfo = await accounts.getAccountInfo({
-  principal: stacksAddress,
-  proof: 0,
-});
+async function getAccountInfo() {
```

Comment:
I would rename this method so it is not the same as the one above, eg `getAccountInfoWithoutProof`

Reply:
Fixed